docs/deploying.adoc

```diff
@@ -26,13 +26,13 @@ virtualClusters:
     targetCluster:
       bootstrap_servers: myprivatecluster:9092  # <2>
     clusterNetworkAddressConfigProvider:
-      type: PortPerBroker                       # <3>
+      type: PortPerBrokerClusterNetworkAddressConfigProvider                       # <3>
       config:
         bootstrapAddress: mypublickroxylicious:9192    # <4>
 ----
 <1> The name of the virtual cluster.
 <2> The bootstrap of the (physical) Kafka Cluster.  This is the Kafka Cluster being proxied.
-<3> The name of a cluster network address config provider. The built-in types are `PortPerBroker` and `SniRouting`.
+<3> The name of a cluster network address config provider. The built-in types are `PortPerBrokerClusterNetworkAddressConfigProvider` and `SniRoutingClusterNetworkAddressConfigProvider`.
```

Comment:
🤮 This is a bit sad exposing those names to users is a bit horrible. I accept thats the cost of avoiding resolvable naming clashes. 

Maybe we can find better names for the underlying classes.

Reply:
Or we could support custom aliases for these cases but not the Filter API? Are we committed to the cluster network address config provider being pluggable by users? It's in the API module.

Reply:
I like those as names! Could you pick up that rename in a new PR @k-wall ?

Reply:
Sure thing!